### PR TITLE
btrfs*: add Chinese translation

### PR DESCRIPTION
**Pull request by CoelacanthusHex**
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

**Comment by bl-ue** [comment]
Hi @CoelacanthusHex, this is great, but this PR will have to wait a little while, see my PR #5240 for normalizing punctuation across all Chinese pages.

**Comment by CoelacanthusHex** [comment]
ok, I am waiting, and I will correct punctuation in my pr as #5240 do when it be merged

**Comment by bl-ue** [comment]
Please feel free to contribute suggestions to that PR, as I don't know Chinese. I'd love for some help from native Chinese speakers ;)

File: pages.zh/linux/btrfs-device.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# btrfs device
+
+> 管理 btrfs 文件系统中的设备。
+> 更多信息可参见: <https://btrfs.wiki.kernel.org/index.php/Manpage/btrfs-device>.
+
+- 将一个或多个设备添加到 b​​trfs 文件系统中:
+
+`sudo btrfs device add {{指向设备1的路径}} [{{指向设备2的路径}}] {{指向 btrfs 文件系统的路径}}`
+
+- 从btrfs文件系统中删除设备:
```
Comment (bl-ue) [inline pages.zh/linux/btrfs-device.md:10]:
```suggestion
- 从 btrfs 文件系统中删除设备:
```

**Review by bl-ue — CHANGES_REQUESTED** [review]
Hi @CoelacanthusHex, I found a little oversight.

Also, will you please:
1. Change all colons (including ones at the end of example descriptions) to Chinese colons (`：`)
2. Change all periods (including ones at the end of more information links) to Chinese periods (`。`)

Thank you!

**Comment by CoelacanthusHex** [comment]
> Hi @CoelacanthusHex, I found a little oversight.
> 
> Also, will you please:
> 
>     1. Change all colons (including ones at the end of example descriptions) to Chinese colons (`：`)
> 
>     2. Change all periods (including ones at the end of more information links) to Chinese periods (`。`)
> 
> 
> Thank you!

Done!

**Comment by bl-ue** [comment]
Great.  I think this PR will be merged soon, as I find all of the punctuation and formatted well.  I especially like what you've done here:
https://github.com/tldr-pages/tldr/pull/5244/files#diff-4a913c56aa1d36868ed6ef85cf4d56a934b94c19d5b650f0e4a90d47fc46fcdfR3,
with the `（COW）`, because that solves an issue I've been having with the Chinese parentheses around English sentences (#5240).

File: pages.zh/linux/btrfs-device.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# btrfs device
+
+> 管理 btrfs 文件系统中的设备。
+> 更多信息可参见：<https://btrfs.wiki.kernel.org/index.php/Manpage/btrfs-device>。
```
Comment (navarroaxel) [inline pages.zh/linux/btrfs-device.md:4]: I think we use this for the link:
```suggestion
>  更多信息：<https://btrfs.wiki.kernel.org/index.php/Manpage/btrfs-device>。
```

Comment (bl-ue) [reply pages.zh/linux/btrfs-device.md:4]: According to Google:
- `更多信息可参见` = `For more information, see` 
- `更多信息` = `More information`

File: pages.zh/linux/btrfs-filesystem.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# btrfs filesystem
+
+> 管理 btrfs 文件系统。
+> 更多信息可参见：<https://btrfs.wiki.kernel.org/index.php/Manpage/btrfs-filesystem>。
```
Comment (bl-ue) [inline pages.zh/linux/btrfs-filesystem.md:4]:
```suggestion
> 更多信息：<https://btrfs.wiki.kernel.org/index.php/Manpage/btrfs-filesystem>。
```

File: pages.zh/linux/btrfs-scrub.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# btrfs scrub
+
+> 清理 btrfs 文件系统以验证数据完整性。
+> 建议每月运行一次 scrub。
+> 更多信息可参见：<https://btrfs.wiki.kernel.org/index.php/Manpage/btrfs-scrub>。
```
Comment (bl-ue) [inline pages.zh/linux/btrfs-scrub.md:5]:
```suggestion
> 更多信息：<https://btrfs.wiki.kernel.org/index.php/Manpage/btrfs-scrub>。
```

File: pages.zh/linux/btrfs-subvolume.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# btrfs subvolume
+
+> 管理 btrfs 子卷和快照。
+> 更多信息可参见：<https://btrfs.wiki.kernel.org/index.php/Manpage/btrfs-subvolume>。
```
Comment (bl-ue) [inline pages.zh/linux/btrfs-subvolume.md:4]:
```suggestion
> 更多信息：<https://btrfs.wiki.kernel.org/index.php/Manpage/btrfs-subvolume>。
```

**Review by bl-ue — CHANGES_REQUESTED** [review]
@CoelacanthusHex please apply @navarroaxel's suggestion and these.

See https://github.com/tldr-pages/tldr/pull/5244#discussion_r571672380.  We prefer the shorter version.

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @CoelacanthusHex!

This reminds me, I should write another page for a btrfs subcommand to keep the ball rolling.

**Comment by sbrl** [comment]
Are we ok to merge here, @bl-ue/

**Comment by bl-ue** [comment]
@sbrl I'd like to wait for #5240.

**Comment by bl-ue** [comment]
Waiting for #5240.  Sorry, I need to get back to that.

**Review by einverne — APPROVED** [review]
Looks good to me. @bl-ue is it good to merge?

**Comment by bl-ue** [comment]
https://github.com/tldr-pages/tldr/pull/5240 soooorry about this, I need to get back to that ☹️

**Comment by bl-ue** [comment]
Okay, yes, #5240 will be my first priority (besides PR reviews) 👍🏻

**Comment by marchersimon** [comment]
How about we merge this PR for now and make changes when we're done with #5240? Many people could use the pages in the meantime.

**Review by bl-ue — APPROVED** [review]
Thank you @CoelacanthusHex!  I'm terrible sorry that this has taken nearly half a year, especially on your first PR 😖

**Review by CleanMachine1 — APPROVED** [review]

**Comment by CleanMachine1** [comment]
Merging.

**Pull request merged**